contents/product-engineers/best-customer-retention-metrics.md

```diff
@@ -0,0 +1,246 @@
+
+A clear understanding of your CLTV also gives you a clear understanding of who your most valuable customers are and how they behave. You can then, for example, work to move more people into that cohort, or work out how to better serve your most loyal customers.
+
+### How to calculate your CLTV
+
+Take the average revenue per user (ARPU) within a specific timeframe, say monthly or annually, and multiply it by your average customer lifespan. 
+
+For example, if your ARPU is $50 per month and your average customer lifespan is 12 months, then your CLTV would be $50 x 12, or $600.
+
+### What’s a good CLTV?
+
+To work this out you need to know your CAC. That’s your customer acquisition cost, or, simply put, the average cost to your business of snagging a new paying customer. If your CAC is lower than your CLTV then you’ve got a healthy business that’s able to acquire new paying customers without losing money. If it’s not, you're in trouble.
+
+As a guide, at early-stage startups the CLTV might be lower as you acquire new customers and find product-market fit. According to data compiled by search optimization firm [First Page Sage](https://firstpagesage.com/seo-blog/the-ltv-to-cac-ratio-benchmark/), the average CLTC to CAC ratio at B2B SaaS companies is 4:1 At B2C SaaS companies it’s 2.5:1.
+
+And this kind of healthiness compounds: acquiring customers without spending a ton on marketing allows you to focus more on improving and expanding your existing product, delighting your most loyal customers, and improving overall customer satisfaction.
+
+## 3. Revenue churn rate
+
+### What is revenue churn rate?
+
+It’s not just customers who churn, revenue churns with them. This metric helps you understand how much money your business makes from each user over time. It can help you spot where you’re losing revenue due to downgrades, people failing to renew, or, worst of all, cancellations. 
+
+While your customer churn rate looks at the number of customers you lose in a given period, your revenue churn specifically focuses on the dollar value of recurring period lost, or gained, in a certain period.
+
+### How is it useful?
+
+Any subscription-based business should be measuring their revenue churn rate. These days, that’s pretty much all businesses from coffee subscription businesses to B2B SaaS companies and everything in between. Businesses that sell big, high-value items (think houses, industrial equipment) likely don’t need to focus on revenue churn.
+
+### How to calculate revenue churn rate
+
+You can calculate your revenue churn rate based on either gross or net revenue. A focus on gross revenue is simpler as it’ll show you how much revenue is churning out of your business, but a focus on net revenue will show both the amount of revenue lost from, say, customers downgrading, but also the revenue gained from customer upgrading.
```

Comment:
```suggestion
You can calculate your revenue churn rate based on either gross or net revenue: 

- **Gross revenue** is simpler but less useful, as it’ll show you how much revenue is churning out of your business without much context. 

- **Net revenue** is harder to calculate, but way more powerful because it'll show both the amount of revenue lost from, say, customers downgrading, but also the revenue gained from customers upgrading.
```